honeybee_radiance/modifier/material/bsdf.py

```diff
@@ -376,13 +376,12 @@ def from_dict(cls, data, folder='.'):
 
         fp = os.path.join(folder, '%s.xml' % data['identifier'])
         # write bytes to xml file
-        cls.decompress_bytes_to_file(data['bsdf_data'], fp)
+        cls.decompress_to_file(data['bsdf_data'], fp)
```

Comment:
This will be an issue when we move the JSON file for the model around. Probably we should think of loading the data to the object and keeping it there until we write it to a folder / file.

Reply:
I think that is probably something that we could implement now that we have set honeybee up in a way that you can use the same instance of a BSDF to assign a modifier to a whole host of objects (so you don't end up duplicating the data in the XML for each object that the BSDF is assigned to). It's a lot to tackle in this PR, though. I will open an issue for it. https://github.com/ladybug-tools/honeybee-radiance/issues/80

Reply:
Sounds good! And I won't be worried about the duplication at this point if that is going to be too much of a hassle.